atf_test/atf/testblocksets/testblockset1.yaml

```diff
@@ -61,14 +61,14 @@ testblock_small:
       root_frame: world
       measured_frame: reference1
       groundtruth:
-        data: 2.0
-        epsilon: 1.9999
+        data: 3.0
```

Comment:
@floweisshardt verify if there is another tf jump bug?

Reply:
Running the tests locally I still get a jerk of around `2.0`. @fmessmer Where did you get a higher jerk?

Reply:
on some local run - it's been a month ago...

Reply:
seems as there are some outliers:
![image](https://user-images.githubusercontent.com/158051/104332613-a4e09d00-54f0-11eb-9de6-63dd0802f42e.png)

this screenshot shows that most of the tests have low jerk but some have higher jerks close to the limit. Needs to be investigated in a new issue:  https://github.com/floweisshardt/atf/issues/165